workers/generic-worker/errorreport/errorreport.go

```diff
@@ -0,0 +1,28 @@
+package errorreport
+
+import (
+	"github.com/taskcluster/taskcluster/v30/internal/workerproto"
+)
+
+func Send(proto *workerproto.Protocol, err error, debugInfo map[string]string) {
+	title := "generic-worker error"
+	description := err.Error()
```

Comment:
A `panic` can send more than just an error, so it might be better to receive `message interface{}` 
rather than `err error` and then format it as a string with e.g. `description := fmt.Sprintf("%s", message)`. This would allow you to call `errorreport.Send` for any panic, rather than just panics that report the `error` type.